src/pages/TriggerReleasePlanPage.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { Helmet } from 'react-helmet';
+import NamespacedPage from '../components/NamespacedPage/NamespacedPage';
+import PageAccessCheck from '../components/PageAccess/PageAccessCheck';
+import { TriggerReleaseFormPage } from '../components/ReleaseService/ReleasePlan/TriggerRelease/TriggerReleaseFormPage';
+import { FULL_APPLICATION_TITLE } from '../consts/labels';
+import { ReleasePlanModel } from '../models';
+import { AccessReviewResources } from '../types';
+
+const TriggerReleasePlanPage: React.FC<React.PropsWithChildren<unknown>> = () => {
+  const accessReviewResources: AccessReviewResources = [
+    { model: ReleasePlanModel, verb: 'create' },
```

Comment:
Shouldn't this be ReleaseModel?
```suggestion
    { model: ReleaseModel, verb: 'create' },
```